rules/windows/builtin/win_user_driver_loaded.yml

```diff
@@ -21,18 +21,18 @@ detection:
         Service: '-'
     selection_2:
         ProcessName|contains:
```

Comment:
that's the place. should be `endswith` instead of `contains`

Reply:
Fixed :)